### DISCUSSION TITLE
Postponement of this Project

### DISCUSSION THREAD

**Discussion by debjitpal5040** [dakshineswarshayaklibrary/website#24] [Announcements]
Due to unavailability of sufficient working members, further work in this project is now postponed as this is a huge project and impossible to continue further by just 3 people. We have also other works to do rather than investing our valuable time in this non-profit project. Until sufficient number of people agree to contribute in this project and really contribute, no further progress will be made in this project from us. 

Signing off 🙂

@debjitpal5040  
@Soumabha-Banerjee 
@airlive11

**Comment by airlive11** [comment]
**I agree with you!!!**

          ---Sakshya Bhattacharya

**Comment by Soumabha-Banerjee** [comment]
Completely agree with @debjitpal5040
